File: Basic Coding Problems/01 Fundamentals/015 Write a program to calculate the area of an equilateral triangle.Write a C program to calculate the area of an equilateral triangle/C++/equilateral_triangle.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    float side, area;

    cout << "Enter the side of the equilateral triangle: ";
    cin >> side;

    area = (sqrt(3) / 4) * side * side;

    cout << "The area of the equilateral triangle is: " << area << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/015 Write a program to calculate the area of an equilateral triangle.Write a C program to calculate the area of an equilateral triangle/Java/equilateral_triangle.java
import java.util.Scanner;

public class equilateral_triangle {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the side of the equilateral triangle: ");
        float side = scanner.nextFloat();

        float area = (float) (Math.sqrt(3) / 4) * side * side;

        System.out.println("The area of the equilateral triangle is: " + area);
    }
}